index.bs

```diff
@@ -5517,12 +5519,21 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
     verify that the [=authData/flags/UV=] bit of the <code>[=flags=]</code> in |authData| is set.
     Otherwise, ignore the value of the [=authData/flags/UV=] [=flag=].
 
+1. If the [=authData/flags/BE=] bit of the <code>[=flags=]</code> in |authData| is not set, verify that the [=authData/flags/BS=] bit is not set.
+
 1. If the credential [=backup state=] is used as part of [=[RP]=] business logic or policy,
     let |currentBe| and |currentBs| be the values of the [=authData/flags/BE=] and [=authData/flags/BS=] bits, respectively,
     of the <code>[=flags=]</code> in |authData|.
     Compare |currentBe| and |currentBs| with
-    <code>|credentialRecord|.[$credential record/backupEligible$]</code> and <code>|credentialRecord|.[$credential record/backupState$]</code>
-    and apply [=[RP]=] policy, if any.
+    <code>|credentialRecord|.[$credential record/backupEligible$]</code> and <code>|credentialRecord|.[$credential record/backupState$]</code>:
+
+    1. If <code>|credentialRecord|.[$credential record/backupEligible$]</code> is set, verify that |currentBe| is set.
+
+    1. If <code>|credentialRecord|.[$credential record/backupEligible$]</code> is not set, verify that |currentBe| is not set.
+
+    1. If <code>|credentialRecord|.[$credential record/backupEligible$]</code> is not set, verify that |currentBs| is not set.
```

Comment:
This can be eliminated now.

```suggestion
```

Reasoning:

 1. Let `be=0 => bs=0` denote the requirement `1. If the [=authData/flags/BE=] bit of the <code>[=flags=]</code> in |authData| is not set, verify that the [=authData/flags/BS=] bit is not set.`.
 2. Let `BE=1 => be=1` denote the requirement `1. If <code>|credentialRecord|.[$credential record/backupEligible$]</code> is set, verify that |currentBe| is set.`.
 3. Let `BE=0 => be=0` denote the requirement `1. If <code>|credentialRecord|.[$credential record/backupEligible$]</code> is not set, verify that |currentBe| is not set.`.
 4. Let `BE=0 => bs=0` denote the requirement `1. If <code>|credentialRecord|.[$credential record/backupEligible$]</code> is not set, verify that |currentBs| is not set.`
 5. Collect equations:
    ```
    be=0 => bs=0
    BE=1 => be=1
    BE=0 => be=0
    BE=0 => bs=0
    ```
    Rearrange:
    ```
    BE=1 => be=1

    BE=0 => be=0
    be=0 => bs=0

    BE=0 => bs=0
    ```
    We see that `BE=0` implies `bs=0` through two paths, i.e. the fourth equation is redundant.

Reply:
Ah, I see what you mean, the authentication response would fail verification earlier when we assert that `be=0` and `bs=1` is bad, before there was a chance for `BE=0` and `bs=1` to assert a failure.

Reply:
I removed the redundant step here: https://github.com/w3c/webauthn/pull/1907/commits/4514f5fcefa2c8d8e23fbd8e8b1cbc3990ce959d